ArmPkg/Drivers/SmmuDxe/IoMmu.h

```diff
@@ -0,0 +1,81 @@
+/** @file IoMmu.h
+
+    This file is the IoMmu header file for SMMU driver.
+
+    Copyright (c) Microsoft Corporation.
+    SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef IOMMU_H_
+#define IOMMU_H_
+
+#include <Library/ArmLib.h>
+#include "SmmuV3.h"
+
+PAGE_TABLE *
+EFIAPI

```

Comment:
If these functions are internal to `SmmuDxe`, they do not need to be `EFIAPI`.

Reply:
removed all EFIAPI except for IoMmu protocol functions as all other functions are internal to this module.